report/main.qmd

```diff
@@ -1 +1,166 @@
+3. Does the effectiveness of different interventions vary between sites?
+
+The structure of the remainder of this report is as follows. Following a brief description about the dataset in Section 2, suggestions on exploratory data analysis (EDA) are presented in Section 3. Section 4 proposes formal analysis methods, and Section 5 summarizes our recommendations. 
+
+## Data Summary
+This is a quasi-experiment study for investigating the effects of five interventions  on consumer inclination towards zero-calorie beverages. The dataset contains the daily sales of different types of beverages (including zero-calorie, sugared and others) under various interventions across three sites, from October 27 to May 23. In this study, the daily sales of each type of beverage are used for measuring the consumers' preference on this type of beverage. The crucial variables are the daily sales of zero-calorie beverages, sugared beverages and total of all the beverages, which are measured as continuous data. On the other hand, based on the objectives of the study, the site and intervention should be treated as categorical variables for further analysis, and the days of week are also recorded as a categorical variable to determine potential impacts of time on the outcomes. Additionally, the dataset includes a time-counting variable that captures the timeline of the entire experiment, indicating that each site has around 220 time points. For the entire dataset, the records of these key variables are relatively complete, with sugared and zero-calorie beverages having nine observations missing. However, records for other types of beverages show a certain proportion of missing data, which could be because they are not the items of interest and excluded in recording.
+
+## EDA Analysis
+Before doing any analysis, it is important to first assess data missingness. A graph (like Figure 1) can be drawn to visualize the proportion of missing values for all columns in the dataset. If there are missing data, it is suggested to create a table to show the data missingness by `Site` for columns that have missing data (see Table 1). To see for each site from which intervention type the missing data comes from, an expanded table can be made to show the data missingness by `Site` and `Intervention` for columns with missing data (see Table A1 in Appendix). For columns that will be used in the analysis to answer the research question, it is important to find out why the data are missing.
+\
+```{r}
+sale_data <- read.table(here("rawdata/june1data.csv"), head=T, sep=",")
+
+# replace "dis " with "dis"
+sale_data <- sale_data %>%
+  mutate(Intervention = replace(Intervention, Intervention == "dis ", "dis"))
+
+#convert DofW, Site and Intervention columns to factor
+sale_data <- sale_data %>% 
+  mutate_at(c('DofW', 'Site', 'Intervention'), as.factor)
+```
+
+```{r missing-data-plot, fig.height=3}
+#missing data, overall
+plot_missing_values_proportions(df=sale_data) +
```

Comment:
I think the x-axis could be labeled differently. "proportion of missing values" sounds like it could mean the proportion of all missing values coming from that column to me. I think something like "proportion of values missing" makes it more clear you are plotting for each variable